public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -21,6 +21,7 @@ import {
   VariableDependencyConfig,
   VizPanel,
 } from '@grafana/scenes';
+import { DataQuery } from '@grafana/schema/dist/esm/index';
```

Comment:
We should be fine to import directly from `'@grafana/schema'` here:

```suggestion
import { DataQuery } from '@grafana/schema';
```